lib/realtime/channels.ex

```diff
@@ -8,32 +8,42 @@ defmodule Realtime.Channels do
 
   import Ecto.Query
 
+  def list_channels(db_conection) do
+    query = from(c in Channel)
+    {query, args} = Repo.to_sql(:all, query)
+
+    db_conection
+    |> Postgrex.query(query, args)
+    |> Repo.result_to_structs(Channel)
```

Comment:
perhaps this "DSL to encoded result" is worth wrapping in a function, as this is the second time similar code has been used

Reply:
good call will do 👍